### ISSUE TITLE
Headers array not sent in ASM telemetry logging anymore 

### ISSUE THREAD

**Issue by oakh1** [F5Networks/f5-telemetry-streaming#127]
### Environment
 * Telemetry Streaming Version: 1.19.0-3
 * BIG-IP Version: 15.1.2.1

### Summary
After upgrading from TS 1.17.0 -> 1.19.0 the client headers (for the blocked ASM log event) doesn't seem to be sent from telemetry streaming. I am using a simple Generic_HTTP consumer. It was working in 1.17.0-3 but after upgrade I only seem to be receiving the headers 1/100 of the times.

### Steps To Reproduce
1. Submit the following declaration:
```json
    "ELK": {
        "class": "Telemetry_Consumer",
        "type": "Generic_HTTP",
        "host": "10.10.10.10",
        "protocol": "http",
        "port": 9550,
        "path": "/",
        "method": "POST",
        "headers": [
            {
                "name": "content-type",
                "value": "application/json"
            }
        ]
    },
```

### Expected Behavior
The client HTTP request headers to be sent just like in 1.17.0

### Actual Behavior
The headers are not included in the ASM blocked event data.

**Comment by dstokesf5** [comment]
Are you seeing problems with the HTTP headers (e.g. "content-type") on the HTTP requests from Telemetry Streaming, or are you referring to a property, or properties, in the payload of the HTTP request?

**Comment by Eken92** [comment]
Sorry for being a bit unclear. The issue is missing properties inside the payload of the http request. The JSON that is sent is valid but the headers property that was present in earlier versions seems to be missing 99 times out of 100. I have tcpdumped the traffic leaving the f5 so there's no issues with our log parsing.

When an event is blocked the ASM engine adds all the client headers to the asm event (that is, the payload where the support ID is included).

**Comment by oakh1** [comment]
After downgrade to 1.17.0 I am now getting all the information inside the ASM logs sent using telemetry again. It also seems like there was a big amount of log events that aren't actually sent either, before downgrade I had around 20-50 events/minute (which I thought was strange from the beginning) but now after downgrade we have around 2-4x more logs received.

Something seems to be really wrong with the ASM logs in the newer version. I have not verified if the issues already started with 1.18.0 as I haven't tested it, but somewhere between 1.17.0 -> 1.19.0.

Thank you :)

**Comment by dstokesf5** [comment]
We received another report that this is affecting 1.18.0 as well. This issue is in our internal product backlog as AUTOTOOL-2451.

**Comment by dstokesf5** [comment]
This issue was resolved as part of the 1.20.0 release.
